samples/word/50-document/manage-settings.yaml

```diff
@@ -0,0 +1,133 @@
+order: 6
+id: word-document-manage-settings
+name: Manage settings
+description: 'This sample shows how to add, get, and delete settings in the file.'
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#create-setting").click(() => tryCatch(createSetting));
+        $("#get-settings").click(() => tryCatch(getSettings));
+        $("#delete-all-settings").click(() => tryCatch(deleteAllSettings));
+
+        async function createSetting() {
+          await Word.run(async (context) => {
+            let key = $("#key")
+              .val()
+              .toString();
+            let value = $("#value")
+              .val()
+              .toString();
+            const settings = context.document.settings;
+            let setting = settings.add(key, value);
```

Comment:
The snippet throws an error for an empty key. Consider adding a check, unless you feel it detracts from the sample too much.